de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/model/DhlCustomDeliveryDataDetail.java

```diff
@@ -46,13 +46,21 @@
 	@Nullable
 	String trackingUrl;
 
+	boolean internationalDelivery;
+
+	//	@Nullable
+	//	DhlCustomsDocument customsDocument;
+
 	@Builder(toBuilder = true)
 	private DhlCustomDeliveryDataDetail(
 			final int packageId,
 			@NonNull final DhlSequenceNumber sequenceNumber,
 			@Nullable final byte[] pdfLabelData,
 			@Nullable final String awb,
-			@Nullable final String trackingUrl)
+			@Nullable final String trackingUrl,
+			final boolean internationalDelivery
+			//			@Nullable final DhlCustomsDocument customsDocument
```

Comment:
agqain, please clean this up, or add a comment explaining the use of this commented out code